test/SILOptimizer/dont_hoist_begin_access_loop_with_no_exit.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %target-swiftc_driver -O %s
+
+// Issue URL
```

Comment:
I assume that you meant:

```suggestion
// https://github.com/apple/swift/issues/67084
```